ecs-cli/modules/utils/logger/logger.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2015-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package logger
+
+import (
+	"os"
+	"github.com/sirupsen/logrus"
+)
+
+func SetupLogger() {
+	logrusConfig()
+}
+
+func logrusConfig() {
+	logrus.SetLevel(logrus.InfoLevel)
```

Comment:
Does this in any way conflict with: https://github.com/aws/amazon-ecs-cli/blob/master/ecs-cli/modules/app.go#L25

Can you test with `--debug` to check?

Reply:
```
~/go/src/github.com/aws/amazon-ecs-cli(fix-logger)$ ecs-cli compose --debug up 
DEBU[0000] Parsing the compose yaml...                  
ERRO[0000] Unable to open ECS Compose Project            error="open docker-compose.yml: no such file or directory"
FATA[0000] Unable to create and read ECS Compose Project  error="open docker-compose.yml: no such file or directory"

```

Also tried it for `ecs-cli compose --debug up` and it works, but output was very...verbose :)